std/path.d

```diff
@@ -58,6 +58,22 @@ module std.path;
 import std.file; //: getcwd;
 import std.range.primitives;
 import std.traits;
+import std.meta : anySatisfy;
```

Comment:
Are selective imports fixed or does this still add a public alias? If the latter, using static import might be better if you want to avoid pollution of symbol lookup in this module.